docs/playbooks/wind-river-cloud-platform-deployment-manager.yaml

```diff
@@ -774,7 +775,7 @@
               register: new_reboot
               when: waiting_after_new_reboot.failed
 
-            when: waiting_after_reboot.failed
+            when: waiting_after_reboot.failed or waiting_after_new_reboot.unreachable
```

Comment:
waiting_after_new_reboot is defined in line767, the condition should not contain a register inside the block.
Please mark as draft before tested

Reply:
Condition was corrected to when: waiting_after_reboot.failed or waiting_after_reboot.unreachable and tested successfully.